code/api/src/migrations/8-shipments.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.createTable('shipments', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'users',
+          key: 'id'
+        },
+        allowNull: false
+      },
+      crateId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'crates',
+          key: 'id'
+        },
+        allowNull: false
+      },
+      deliveryDate: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
```

Comment:
I wish I knew more of the back-end and what I was looking at here… Is this our mutator that is going to allow our user to update their date?